Week3/homework/6-step3.js

```diff
@@ -4,3 +4,17 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 // add your solution here, or add a comment on how you would tackle this problem
+//arr2d
+for (let i = 0; i < arr2d.length; i++) {
+  for (let j = 0; j < arr2d[i].length; j++) {
+    console.log(arr2d[i][j]);
+  }
+}
+
+//arr3d
+for (let i = 0; i < arr3d.length; i++) {
+  for (let j = 0; j < arr3d[i].length; j++) {
+    for (let k = 0; k < arr3d[i][j].length; k++)
+      console.log(arr3d[i][j][k]);
+  }
```

Comment:
Well done, this does tackle the 2 and 3 dimensional array.
And how would you handle a 4-dimensional array? This won't be able to handle it.
And a 5, 6, 7, or 8 dimensional array?
The best answer to handle arrays with an unknown number of dimensions is to use something called recursion.
http://eloquentjavascript.net/03_functions.html#h_jxl1p970Fy